newton/tests/test_hydroelastic.py

```
@@ -0,0 +1,510 @@
+    def _run_viewer_test(self, shape_type: ShapeType, solver_name: str = "xpbd", cube_half: float = CUBE_HALF_LARGE):
+        device = wp.get_device("cuda:0")
+        solver_fn = solvers[solver_name]
+
+        model, solver, state_0, state_1, control, collision_pipeline, _, _ = build_stacked_cubes_scene(
+            device, solver_fn, shape_type, cube_half
+        )
+
+        try:
+            viewer = newton.viewer.ViewerGL()
+            viewer.set_model(model)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)